src/test/java/core/basesyntax/ConverterTest.java

```diff
@@ -0,0 +1,87 @@
+package core.basesyntax;
+
+import core.basesyntax.converter.DataConverter;
+import core.basesyntax.converter.DataConverterImpl;
+import core.basesyntax.model.FruitTransaction;
+import java.util.List;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class ConverterTest {
+    private DataConverter dataConverter;
+
+    @BeforeEach
+    void setUp() {
+        dataConverter = new DataConverterImpl();
+    }
+
+    @Test
+    void converter_testConverterToTransactions_ok() {
```

Comment:
```suggestion
    void converter_validTransactions_ok() {
```
second part is about given conditions